docs/static/config-details.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[jvm-settings]]
 === JVM settings
 
-Configure the jvm settings in the `jvm.options` <<settings-files,settings file>>.
+Configure the jvm settings in the `jvm.options` <<settings-files,settings file>> or with the <<ls-java-opts, `LS_JAVA_OPTS`>>.
```

Comment:
I'm not sure 

```suggestion
Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable
```

Reply:
Far better, my `or` wasn't an exclusive or, but this way sounds more explicit!